Source/Scene/Scene.js

```diff
@@ -822,6 +822,19 @@ define([
         },
 
         /**
+         * Gets an event that's raised when the terrain provider is changed
+         * @memberof Scene.prototype
+         *
+         * @type {Event}
+         * @readonly
+         */
+        terrainProviderChanged : {
```

Comment:
Add to CHANGES.md.  You could also make the `Globe` event private and only document this.